app/src/main/java/fr/free/nrw/commons/filepicker/UploadableFile.java

```diff
@@ -118,6 +119,27 @@ private DateTimeWithSource getFileCreatedDateFromCP(Context context) {
         }
     }
 
+    /**
+     * Indicate whether the EXIF contains the location (both latitude and longitude).
+     *
+     * @return whether the location exists for the file's EXIF
+     */
+    public boolean hasLocation() {
+        try {
+            ExifInterface exif = new ExifInterface(file.getAbsolutePath());
+            final String latitude = exif.getAttribute(ExifInterface.TAG_GPS_LATITUDE);
+            final String longitude = exif.getAttribute(ExifInterface.TAG_GPS_LATITUDE);
+            return latitude != null && longitude != null;
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Please output to the logging facility used by other places, so that error appear in logcat. Same below. Thanks! :-)

Reply:
Do you mean SLF4J as the logging facility?
![Logger usage](https://github.com/commons-app/apps-android-commons/assets/53111300/d9f22129-5ac1-40d3-bf3d-33145b3f13a5)
From what I observe SLF4J is indeed implemented in the `logging` package but there hasn't been similar use of `Logger` instances or `getLogger` method yet. So I'm not sure if [0f4bb8f](https://github.com/commons-app/apps-android-commons/pull/5343/commits/0f4bb8f9e2b90ec804e4b84d2800425599ae92ab) is the correct approach. If so, I've also applied the same thing for `getDateTimeFromExif()`, the method in the `UploadableFile` class under `hasLocation`.